examples/xonotic/gameserver.yaml

```diff
@@ -24,5 +24,5 @@ spec:
     spec:
       containers:
         - name: xonotic
-          image: us-docker.pkg.dev/agones-images/examples/xonotic-example:1.2
+          image: us-docker.pkg.dev/agones-images/examples/xonotic-example:1.3
```

Comment:
It looks like in https://github.com/googleforgames/agones/pull/3027 we went from 0.9 to 1.0 instead of to 0.10. 

Now that we are at 1.2 we should just keep bumping the minor version, so going from 1.2 to 1.3 seems reasonable.